src/backend/distributed/shared_library_init.c

```diff
@@ -754,6 +755,20 @@ RegisterCitusConfigVariables(void)
 		GUC_STANDARD,
 		NULL, NULL, NULL);
 
+	DefineCustomBoolVariable(
```

Comment:
Let's also add a comment here to remove this GUC when we stop supporting PG 12, should contain `PG_VERSION_NUM < 120000` somewhere for easy grepping.